projects/ngx-datatable/src/lib/components/body/body.component.ts

```diff
@@ -64,145 +67,181 @@ import {
+                    [isSelected]="selector.getRowSelected(group)"
+                    [innerWidth]="innerWidth"
+                    [offsetX]="offsetX"
+                    [columns]="columns"
+                    [rowHeight]="getRowHeight(group)"
+                    [row]="group"
+                    [rowIndex]="getRowIndex(group)"
+                    [expanded]="getRowExpanded(group)"
+                    [rowClass]="rowClass"
+                    [displayCheck]="displayCheck"
+                    [treeStatus]="group?.treeStatus"
+                    [ghostLoadingIndicator]="ghostLoadingIndicator"
+                    [draggable]="rowDraggable"
+                    [verticalScrollVisible]="verticalScrollVisible"
+                    (treeAction)="onTreeAction(group)"
+                    (activate)="selector.onActivate($event, indexes.first + i)"
+                    (drop)="drop($event, group, rowElement)"
+                    (dragover)="dragOver($event, group)"
+                    (dragenter)="dragEnter($event, group, rowElement)"
+                    (dragleave)="dragLeave($event, group, rowElement)"
+                    (dragstart)="drag($event, group, rowElement)"
+                    (dragend)="dragEnd($event, group)"
+                  >
+                  </datatable-body-row>
+                }
+              }
+
+              <ng-template #bodyRow>
+                @if (isRow(group)) {
+                  <datatable-body-row
+                    role="row"
+                    tabindex="-1"
+                    #rowElement
+                    [disable$]="rowWrapper.disable$"
+                    [isSelected]="selector.getRowSelected(group)"
+                    [innerWidth]="innerWidth"
+                    [offsetX]="offsetX"
+                    [columns]="columns"
+                    [rowHeight]="getRowHeight(group)"
+                    [row]="group"
+                    [rowIndex]="getRowIndex(group)"
+                    [expanded]="getRowExpanded(group)"
+                    [rowClass]="rowClass"
+                    [displayCheck]="displayCheck"
+                    [treeStatus]="group?.treeStatus"
+                    [ghostLoadingIndicator]="ghostLoadingIndicator"
+                    [draggable]="rowDraggable"
+                    [verticalScrollVisible]="verticalScrollVisible"
+                    (treeAction)="onTreeAction(group)"
+                    (activate)="selector.onActivate($event, indexes.first + i)"
+                    (drop)="drop($event, group, rowElement)"
+                    (dragover)="dragOver($event, group)"
+                    (dragenter)="dragEnter($event, group, rowElement)"
+                    (dragleave)="dragLeave($event, group, rowElement)"
+                    (dragstart)="drag($event, group, rowElement)"
+                    (dragend)="dragEnd($event, group)"
+                  >
+                  </datatable-body-row>
+                }
+              </ng-template>
```

Comment:
Can we delete this? I think this is what we have above in the else case.

Reply:
No it is used to support applying drag drop or other custom directives on rows by consumer.